operations/companies.md

```diff
@@ -149,12 +161,28 @@ Returns all company profiles of the enterprise, possibly filtered by identifiers
 | `Contact` | string | optional | Other contact details, such as telephone, email or similar. |
 | `Notes` | string | optional | Additional notes. |
 | `Options` | [Company options](#company-options) | required | Options of the company. |
+| `Department` | string | optional | The internal segmentation of a company, f.e. sales department. |
+| `DunsNumber` | string | optional | The Dun & Bradstreet unique 9-digit DUNS number. |
+| `CreditRating` | [Credit rating](#credit-rating) | optional | Basic credit rating to define creditworthiness of the company. |
 
 #### Company options
 
 | Property | Type | Contract | Description |
 | :-- | :-- | :-- | :-- |
 | `Invoiceable` | boolean | required | Whether the company is invoiceable or not. |
+| `AddFeesToInvoices` | boolean | required | Whether the company has an additional fee applied for invoicing or not. |
+
+#### Credit rating
+
+| Property | Type | Contract | Description |
+| :-- | :-- | :-- | :-- |
+| `Basic` | string [Credit rating basic](#credit-rating-basic)| optional | The level of creditworthiness of the company. |
+
+#### Credit rating basic
+
+* `CreditOk`
+* `PaymentRequiredUpfront`
+* `LocalDecisionRequired`
```

Comment:
Are you absolutely sure no other state can be there? We basically have two types of enums - ones where there is ellipsis at the end specifying that some other value is possible and others without it (usually business critical like activity state).

Reply:
Yes, this reflects an enum with just three values. We will rework and extend credit rating in the future but not in the near future. This is what is needed to unblock NCH @mullerda confirm?

Reply:
Yes. Only these 3 values can ever be there. Or it can be empty. It's not a mandatory property.